src/storage/src/hummock/compactor/mod.rs

```diff
@@ -344,6 +346,9 @@ impl Compactor {
                 _ = &mut shutdown_rx => {
                     tracing::warn!("Compaction task cancelled externally:\n{}", compact_task_to_string(&compact_task));
                     task_status = TaskStatus::ManualCanceled;
+                    for abort_handle in abort_handles {
+                        abort_handle.abort();
+                    }
                     break;
                 }
                 future_result = buffered.next() => {
```

Comment:
Can we abort futures when meet other error ?

IMO, any of the splits fail, we can abort the whole task